### ISSUE TITLE
Add tags attribute for the aws_ses_configuration_set resource

### ISSUE THREAD

**Issue by gmx-git** [hashicorp/terraform-provider-aws#25670]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Please add the 'tags' attribute for the 'aws_ses_configuration_set' resource.

### New or Affected Resource(s)

* aws_ses_configuration_set

### Potential Terraform Configuration

```hcl
resource "aws_ses_configuration_set" "test" {
  name = "ses-configuration-set-test"
  tags = {
    name        = "ses-configuration-set-test"
    environment = "dev"
  }
}
```

### References

* #0000

**Comment by kamilturek** [comment]
The SES API v1 does not allow setting tags on configuration sets. As a workaround, you can use the `aws_sesv2_configuration_set` which allows you to manage the same resource and supports tags.

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sesv2_configuration_set

**Comment by kamilturek** [comment]
I'd say it can be closed via #27056.

**Comment by kamilturek** [comment]
Hey @justinretzolk, could we close this issue?
